crates/next-custom-transforms/src/transforms/server_actions.rs

```diff
@@ -213,6 +215,11 @@ struct ServerActions<C: Comments> {
 }
 
 impl<C: Comments> ServerActions<C> {
+    fn emit_error(&mut self, error_kind: ServerActionsErrorKind) {
```

Comment:
hmmm i can't help but feel that we're reinventing the wheel here somehow. isn't this tracked somewhere already? is there really no way to short-circuit our way out of a visitor by returning a `Err(Nope)` or something?